Hcal/include/Hcal/Event/HcalHit.h

```diff
@@ -193,9 +213,12 @@ class HcalHit : public ldmx::CalorimeterHit {
     isX_ = isX;
   }
 
-  double getPosition() const { return position_; }
-  int getIsX() const { return isX_; }
-  double getTimeDiff() const { return timeDiff_; }
+  /**
```

Comment:
It would be better to use a bool if it has only two values. Otherwise we should extend it to X,Y and Z.

Reply:
OK, I did it for all directions